raiden_contracts/data/source/raiden/TokenNetwork.sol

```diff
@@ -859,12 +889,12 @@ contract TokenNetwork is Utils {
     {
         // Channel represented by channel_identifier must be settled and
         // channel data deleted
-        require(channel_identifier != getChannelIdentifier(receiver, sender));
+        // require(channel_identifier != getChannelIdentifier(receiver, sender), "Bad channel_id");
 
         // After the channel is settled the storage is cleared, therefore the
         // value will be NonExistent and not Settled. The value Settled is used
         // for the external APIs
-        require(channels[channel_identifier].state == ChannelState.NonExistent);
+        require(channels[channel_identifier].state == ChannelState.NonExistent, "Channel still exists");
```

Comment:
Does this check make sense with `NonExistent` meaning open as well?

Reply:
Yes, that check is questionable. It only checks for the channel does not have a settlement in progress. I'll have to revisit this when I have added support for settling multiple times.